jnosql-mapping/jnosql-mapping-column/src/test/java/org/eclipse/jnosql/mapping/column/ColumnTemplateTest.java

```diff
@@ -34,7 +33,7 @@
 @AddPackages(value = {Converters.class, EntityConverter.class, ColumnTemplate.class})
 @AddPackages(MockProducer.class)
 @AddPackages(Reflections.class)
-@AddExtensions({EntityMetadataExtension.class, ColumnExtension.class})
+@AddExtensions({ColumnExtension.class})
```

Comment:
Even at the package on reflection, I would keep the `Extension` mainly because Quarkus or any framework that will avoid reflection will use the JNosQL lite instead.

Reply:
Yeah... I'll keep the `Extension` at `jnosql-mapping-reflection` module.